ada/language/ast.py

```diff
@@ -7593,8 +7593,15 @@ def zip_with_params():
         return params.then(
             lambda _: Self.match_formals(params, Entity, is_dottable_subp).map(
                 lambda m:
-                ParamActual.new(param=m.formal.name.as_bare_entity,
-                                actual=m.actual.assoc.expr),
+                ParamActual.new(
```

Comment:
While that certainly works, I wonder if changing  SingleFormal.name to store an entity wouldn't be a cleaner fix. What do you think ?